.github/workflows/publish-img.yaml

```diff
@@ -88,6 +99,10 @@ jobs:
           build-args: |
             KUBEVIRT_VERSION=${{ env.KUBEVIRT_VERSION }}
 
+      - name: Template branch manifests
+        if: github.repository_owner == 'kubevirt' && github.ref_type == 'branch'
+        run: IMG=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.branch_tag }} make build-installer
```

Comment:
why doesn't this re-use the stanza below ? i.e. template release manifests is already doing this for tags. 
Now it seems you want that also for a specific branch.